solidity/random-beacon/test/RandomBeacon.GroupCreation.test.ts

```diff
@@ -2721,6 +2721,41 @@ describe("RandomBeacon - Group Creation", () => {
       await restoreSnapshot()
     })
   })
+
+  describe("selectGroup", async () => {
+    context("when dkg was not triggered", async () => {
+      before(async () => {
+        await createSnapshot()
+      })
+
+      after(async () => {
+        await restoreSnapshot()
+      })
+
+      it("should revert", async () => {
+        await expect(randomBeacon.selectGroup()).to.be.revertedWith(
+          "Sortition pool unlocked"
+        )
+      })
+    })
+
+    context("when dkg was triggered", async () => {
+      before(async () => {
+        await createSnapshot()
+
+        await genesis(randomBeacon)
+      })
+
+      after(async () => {
+        await restoreSnapshot()
+      })
+
+      it("should select a group", async () => {
+        const selectedGroup = await randomBeacon.selectGroup()
+        expect(selectedGroup.length).to.eq(constants.groupSize)
+      })
```

Comment:
Can we also assert it is the same group that would be returned by the sortition pool if we provided there DKG seed manually?

Reply:
[2e95d35](https://github.com/keep-network/keep-core/pull/2936/commits/2e95d354d3be25ae8ece0448ce352e62c2dd80ce)